armi/reactor/reactors.py

```diff
@@ -293,15 +292,26 @@ def powerMultiplier(self):
 
     @property
     def lib(self):
-        """"Get the microscopic cross section library."""
-        if self._lib is None:
-            runLog.info("Loading microscopic cross section library ISOTXS")
+        """
+        Return the microscopic cross section library.
+
+        Notes
+        -----
```

Comment:
Probably shouldnt be in Notes, as it is fundamental to what this is doing. I think this would be better as a bulleted list; something like:
 - if there is a library currently associated with the Core, it will be returned
 - otherwise, an "ISOTXS" file will be be searched for in the working directory, opened as an ISOTXS object and returned,
 - otherwise, None will be returned